import_install_example.sh

```diff
@@ -4,6 +4,21 @@ default_library_name='shtdlib.sh'
 default_base_download_url='https://raw.githubusercontent.com/sdelements/shtdlib/master'
 default_install_path='/usr/local/bin'
 
+# Temporary debug function
+type -t import | grep -q '^function$' || function debug { echo "${@:2}" ; }
+
+# Import or source
+function import_or_source {
+    if type -t import | grep -q '^function$' ; then
```

Comment:
... but why? 

Can you add a comment block to explain why we prefer `import` over `source` when available?

Reply:
Better?